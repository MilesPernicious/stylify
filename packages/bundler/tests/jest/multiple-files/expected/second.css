._snhbehg{
	content:'second'
}
._vknnqtm{
	color:yellow
}
._vaucfh7{
	content:'Nette'
}
._plqk75e{
	font-size:24px
}
._kjj6q9i{
	font-size:25px
}
._pq6cmjh{
	content:'Symfony'
}
._cpkr460{
	content:'Vue'
}
._gibm8m7{
	content:'vue-script'
}
._uc2e88v{
	font-size:26px
}
._3tngtkf{
	content:'vuejs component'
}
._actjmvl{
	font-size:48px
}