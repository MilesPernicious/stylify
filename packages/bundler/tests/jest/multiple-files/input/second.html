<!--
	@stylify-files[second/nette.latte second/symfony.twig second/vue.vue]
	@stylify-pregenerate[content:,,second,,]
-->

<div class="color:yellow"></div>
