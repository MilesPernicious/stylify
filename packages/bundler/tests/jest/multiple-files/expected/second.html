<!--
	@stylify-files[second/nette.latte second/symfony.twig second/vue.vue]
	@stylify-pregenerate[content:,,second,,]
-->

<div class="_vknnqtm"></div>
